includes/RequestWiki/RequestWikiQueuePager.php

```diff
@@ -16,6 +16,8 @@
 
 class RequestWikiQueuePager extends TablePager {
 
+	public $mDefaultDirection = IndexPager::DIR_ASCENDING;
```

Comment:
This can't have an explicit type hint:
`Fatal error: Type of Miraheze\CreateWiki\RequestWiki\RequestWikiQueuePager::$mDefaultDirection must not be defined (as in class MediaWiki\Pager\IndexPager) in /var/www/html/w/extensions/CreateWiki/includes/RequestWiki/RequestWikiQueuePager.php on line 17`

Reply:
I think that this breaks sorting. As in if someone clicks the column name, it won’t then do the opposite. Instead you just flip it things around

‘’’
		if ( $context->getRequest()->getText( 'sort', 'cw_timestamp' ) == 'cw_timestamp' ) {
			$this->mDefaultDirection = IndexPager::DIR_ASCENDING;
		} else {
			$this->mDefaultDirection = IndexPager::DIR_ DESCENDING;
		}
’’’

Reply:
What do you mean?

https://github.com/user-attachments/assets/faa7c1f9-f500-456e-81f3-43d60a893e21